guides/v2.1/cloud/docker/docker-config.md

```diff
@@ -99,11 +99,17 @@ You must have the following software installed on your local workstation:
     {: .bs-callout .bs-callout-info}
     For `{{site.data.var.ct}}` v2002.0.12, install Magento with the `docker-compose run cli magento-installer` command.
 
+1.  Add Docker host line to hosts file `/etc/hosts`
```

Comment:
Should this be a prerequisite instead of part of the launching instructions? I ask because it seems like a one-time update. You wouldn't typically do this more than once per workstation.